tgui/packages/tgui/layouts/NtosWindow.js

```diff
@@ -17,6 +17,7 @@ export const NtosWindow = (props, context) => {
   } = props;
   const { act, data } = useBackend(context);
   const {
+    PC_device_theme,
```

Comment:
Please override the theme here, in NtosWindow based on this variable, not in every single interface.

Reply:
You can do it like this (allow overriding with a "theme" prop, otherwise default to PC_device_theme):
```js
<Window
  theme={theme || PC_device_theme }
  resizable={resizable}>
```

Reply:
Done.

Reply:
I think this has been solved via explaining my intent (style, if not, let me know)